android/src/main/res/base/anim/rns_slide_in_from_left_ios.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<translate xmlns:android="http://schemas.android.com/apk/res/android"
+    android:duration="@android:integer/config_shortAnimTime"
+    android:fromXDelta="-30%"
```

Comment:
Why we're setting `-30%` here and not just `-100%`? I've just tested how the animation looks like on `fromXDelta="-100%"` and to be honest I cannot spot the difference there 🤔 Am I missing something?

Reply:
The bottom screen doesn't go away fully on iOS:

https://github.com/software-mansion/react-native-screens/assets/5978212/a24e9357-6e0d-4b25-adda-861f6cfdb61a

I captured few frames and it goes off roughly by `30%`